spec/requests/mentees_request_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "Mentees", type: :request do
+  headers = nil
+  before do
+    User.create(first_name: "Julius", last_name: "Ngwu", email: "julius@gmail.com", password: "julius@1", user_type: "mentee")
```

Comment:
Also, there's a FactoryBot for user.
Which will reduce the pain of writing your name and email each time.

But still fine though

Reply:
Implemented. Thanks